apps/server/src/modules/provisioning/strategy/sanis/sanis-response.mapper.ts

```diff
@@ -79,6 +79,16 @@ export class SanisResponseMapper {
 		return RoleMapping[source.personenkontexte[0].rolle];
 	}
 
+	public static mapToGroupNameArr(groups: SanisGruppenResponse[]): string[] {
```

Comment:
Arr is an abrevation we are not using usually. Fo indicating, that it is an array we normally use ...List :)